src/Compute/Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1

```diff
@@ -4944,4 +4944,60 @@ function Test-VirtualMachineScaleSetSecurityTypeUpdate
         # Cleanup
         Clean-ResourceGroup $rgname;
     }
-}
\ No newline at end of file
+}
+
+
+<#
+.SYNOPSIS
+    create a VMSS in flexible mode 
+    Test Attach scenario of adding a vmss
+    Test Detach scenario of removing the VMSS
+#>
+function Test-VirtualMachineScaleSetAttachAndDetach
+{
+    # Setup
+    $rgname = Get-ComputeTestResourceName
+
+    try
+    {
+        # Common
+        $loc = Get-ComputeVMLocation;
+        New-AzResourceGroup -Name $rgname -Location $loc -Force;
+
+        # New VMSS Parameters
+        $vmssName = 'vmssAttachAndDetach' + $rgname;
+        $vmName = 'vm' + $rgname;
+
+        $adminUsername = 'Foo12';
+        $adminPassword = $PLACEHOLDER;
+        $securePassword = ConvertTo-SecureString $adminPassword -AsPlainText -Force;
+        $cred = New-Object System.Management.Automation.PSCredential ($adminUsername, $securePassword);
+
+        $VmssFlex = New-AzVmss `
+          -ResourceGroupName $rgname `
+          -Name $vmssName `
+          -OrchestrationMode 'Flexible' `
+          -Location 'eastus' `
+          -Credential $cred `
+          -DomainNameLabel "scaleset-70f699"
```

Comment:
better to not have this hardcoded, rather do something like d + $rgname.

Reply:
Does not affect the test in this case.